internal/utils/misc.go

```diff
@@ -157,6 +158,43 @@ var (
 	ErrNotRunning  = errors.Errorf("%s is not running.", Aqua("supabase start"))
 )
 
+func GetSeedsFilepaths() string {
+	if seedPaths, _ := GetSeedFiles(afero.NewOsFs()); seedPaths != nil {
+		return fmt.Sprintf("%v", seedPaths)
+	}
+	return DefaultSeedDataPath
+}
+
+/*
+** Match the glob patterns from the config to get
+** a deduplicated array of all migrations files to apply
+** in the declared order
+ */
+func GetSeedFiles(fsys afero.Fs) ([]string, error) {
+	seedPaths := Config.Db.Seed.Path
+	fileSet := make(map[string]struct{})
+	var files []string
+	for _, pattern := range seedPaths {
+		fullPattern := filepath.Join(SupabaseDirPath, pattern)
+		matches, err := afero.Glob(fsys, fullPattern)
+		if err != nil {
+			return nil, errors.Errorf("failed to apply glob pattern for %w", err)
+		}
+		sort.Strings(matches)
+		// Dedup the new matches with previously matched seeds to avoid duplication
+		// between matching patterns
+		for _, match := range matches {
+			if _, exists := fileSet[match]; !exists {
+				fileSet[match] = struct{}{}
+				files = append(files, match)
+			} else {
+				fmt.Fprintf(GetDebugLogger(), "Duplicate seed file found skipping: %s\n", match)
+			}
+		}
+	}
+	return files, nil
```

Comment:
```suggestion
	return utils.RemoveDuplicates(files), nil
```

there's a utils for this to reduce some code duplication.